config/oclcts-authorities.yml

```diff
@@ -0,0 +1,24 @@
+url-pattern:
```

Comment:
This file was produced by the generator. I am not sure how it gets used yet, but would prefer to keep it around for now to not forget about it later.